File: p6/python/p6.py
# GLOBAL VARIABLES
limit = 100

# MAIN PROGRAM
if __name__ == "__main__":
    sum = 0
    for i in range(1, limit):
        for j in range(i + 1, limit + 1):
            sum += i * j

    sum *= 2
    print(sum)

File: p6/c/p6.c
/* IMPORT LIBRARIES */
#include <stdio.h>

/* GLOBAL VARIABLES */
#define LIMIT 100

/* MAIN PROGRAM */
int main(void)
{
	int sum = 0;
	int i, j;
	for(i = 1; i < LIMIT; i++)
		for(j = i + 1; j <= LIMIT; j++)
			sum += i * j;

	sum *= 2;
	printf("Sum: %d\n", sum);

	return 0;
}


